hudi-client/hudi-client-common/src/test/java/org/apache/hudi/utils/HoodieWriterClientTestHarness.java

```diff
@@ -255,6 +255,7 @@ public HoodieWriteConfig.Builder getConfigBuilder(String schemaStr, HoodieIndex.
         .withStorageConfig(HoodieStorageConfig.newBuilder().hfileMaxFileSize(1024 * 1024).parquetMaxFileSize(1024 * 1024).orcMaxFileSize(1024 * 1024).build())
         .forTable(RAW_TRIPS_TEST_NAME)
         .withIndexConfig(HoodieIndexConfig.newBuilder().withIndexType(indexType).build())
+        .withMarkersTimelineServerBasedBatchIntervalMs(10)
```

Comment:
In many places in the PR, the batch interval is decreased from the default of 50ms to 10ms. This shaves the overhead of each write helping trim the runtime of our tests that execute a series of writes.

Reply:
we need some sort of test config defaults where we set some of these defaults in one place so that all tests will be able to use those. 
not for this patch, but just saying in general.